docs/release_notes/0.0.37.md

```diff
@@ -3,6 +3,10 @@
 ## Features
 
 ## Bugfixes
+7e7f15b Fix erroneous remote branch existence check
 
 ## Other
-Refactor deletion of ssm parameters
\ No newline at end of file
+575ea78 Improve feedback from apply cluster on successful sync
```

Comment:
In case you do some more pushes, careful not to rebase and force push anymore as these commit ids will be wrong (I thiink).